javascripts/discourse/components/custom-footer.hbs

```diff
@@ -38,6 +39,7 @@
             </div>
           {{/each}}
         </div>
+        <PluginOutlet @name="easy-footer-second-box-bottom" />
```

Comment:
Instead of adding two outlets here. Perhaps we can just use a plugin outlet wrapper? With that, you can either add a template with the `__after` or `__before` postfix or use the `api. renderBeforeWrapperOutlet ` or `api. renderAfterWrapperOutlet` APIs.